### ISSUE TITLE
off by 1 error in step 6

### ISSUE THREAD

**Issue by isaacovercast** [dereneaton/ipyrad#405]
This is with a reference assembly. 0-based and 1-based nonsense with pysam and bedtools and vcf. Nightmare. From gitter @tim-oconnor:

I can get through steps 1-5 on v0.9.53 with reference-based 3RAD assembly. However it looks like there may be another off-by-one error in step 6:
```
  Step 6: Clustering/Mapping across samples 
  [####################] 100% 0:00:07 | concatenating bams     
  [####################] 100% 0:00:03 | fetching regions       
  [####################] 100% 0:00:11 | building database    
  Encountered an Error.
  Message: ValueError: cannot copy sequence with size 94 to array axis with dimension 95
```

**Comment by leonardslog** [comment]
I was just about to compose an issue for this! I recently encountered the same with a ddrad reference assembly shortly after updating my ipyrad conda virtual environment. I attempted this on two different Ubuntu distributions with one of them being a fresh install in case any of those details help you troubleshoot.

**Comment by isaacovercast** [comment]
Ahhh, this is annoying. I think it'll be a quick fix. I'll try to get to it
tomorrow.

**Comment by isaacovercast** [comment]
Fixed 81d2fcefa103a4d379ac14ea25146fee6f923b34
